### DISCUSSION TITLE
Created a video editor using fabric

### DISCUSSION THREAD

**Discussion by AmitDigga** [fabricjs/fabric.js#9094] [Show and tell]
Project Link: https://github.com/AmitDigga/fabric-video-editor

Still under development. Uses nextjs, mobx, typescript

What do you think?  I will add animations and video filters next.

https://github.com/fabricjs/fabric.js/assets/7884106/f41c3488-7142-4c2e-8047-4d0c16e8aeb4

**Comment by Ayush0Chaudhary** [comment]
This awesome

**Comment by mhieupham1** [comment]
awesome
